src/hotspot/share/prims/jvmtiTagMapTable.cpp

```diff
@@ -252,7 +252,7 @@ void JvmtiTagMapTable::rehash() {
         p = entry->next_addr();
       }
       // get next entry
-      entry = (JvmtiTagMapEntry*)HashtableEntry<WeakHandle, mtServiceability>::make_ptr(*p);
```

Comment:
Nice to get rid of this.  It would be nice if the hashtables didn't need to declare bucket() and bucket_addr() with all these casts but some template nonsense makes it not compile.